tests/vcf/test_vcf.py

```diff
@@ -1,26 +1,17 @@
-import cloudpickle
 import numpy as np
 import pytest
 
 import tiledb.cloud.vcf as vcf
 import tiledb.cloud.vcf.vcf_toolbox as vtb
 
-# Pickle the vcf module by value, so tests run on the latest code.
-cloudpickle.register_pickle_by_value(vcf)
-
-
-# Run VCF tests with:
-#   pytest -m vcf --run-vcf -n 8
-
```

Comment:
Are we sure that we want to remove this one?  I was actually thinking the opposite of introducing it to all ingestors. It is a nice hack by @gspowley. I will add him as a reviewer.

Reply:
It looks like the changes in #683 are reverted in this PR. Maybe those changes need to be merged into this branch?